tests/zfs-tests/include/libtest.shlib

```diff
@@ -3349,13 +3349,23 @@ function get_min
 	echo $min
 }
 
-#
-# Generate a random number between 1 and the argument.
-#
-function random
-{
-        typeset max=$1
-        echo $(( ($RANDOM % $max) + 1 ))
+# A simple function to get a random number between two bounds
```

Comment:
Including this in `tests/zfs-tests/include/math.shlib` would be slightly more appropriate.  Can you also update the comment to make it clear the bounds are inclusive.